README.md

```diff
@@ -2408,9 +2408,9 @@ _Libraries and tools for templating and lexing._
 - [extemplate](https://github.com/dannyvankooten/extemplate) - Tiny wrapper around html/template to allow for easy file-based template inheritance.
 - [fasttemplate](https://github.com/valyala/fasttemplate) - Simple and fast template engine. Substitutes template placeholders up to 10x faster than [text/template](https://golang.org/pkg/text/template/).
 - [gomponents](https://www.gomponents.com) - HTML 5 components in pure Go, that look something like this: `func(name string) g.Node { return Div(Class("headline"), g.Textf("Hi %v!", name)) }`.
-- [gospin](https://github.com/m1/gospin) - Article spinning and spintax/spinning syntax engine, useful for A/B, testing pieces of text/articles and creating more natural conversations.
```

Comment:
Can you provide the reason for removing this project !!!.

Reply:
@yassinebenaid I did in the README!

```
-[x] I removed the following packages around my addition: (please give a short reason for each removal)
gospin - not updated in over 5 years, very little usage
```

Reply:
Ohhh  sorry, didn't notice. Thanks.

Reply:
no worries, thanks for merging!